### ISSUE TITLE
Form labels are not linked to their form control counterparts

### ISSUE THREAD

**Issue by seanforyou23** [syndesisio/syndesis#1003]
This is an accessibility concern, as users who rely on assistive technology will not see that a given label is associated with its corresponding input. Labels should be linked by way of the for html attribute or by nesting the input inside the label element.

A simple way to test this is to click on a label, focus should automatically be placed in the input that the label describes. When this is the case, basic accessibility has been achieved.
